app/models/api/v3/converter_stats_presenter.rb

```diff
@@ -31,7 +32,7 @@ def initialize(key, gql)
       end
 
       def as_json(*)
-        ATTRIBUTES.each_with_object({}) do |attr_key, hash|
+        @graph_attributes.each_with_object({}) do |attr_key, hash|
```

Comment:
These attributes are coming straight from the request without any validation, and then being executed by `public_send`! Luckily not much damage can be done with `public_send` and no arguments, but it's still risky.

Check that each attribute is permitted before executing:

```ruby
def as_json(*)
  unless @graph_attributes.all? { |key| PERMITTED_ATTRIBUTES.include?(key) }
    fail "Not gonna happen!"
  end

  @graph_attributes.each_with_object({}) do |attr_key, hash|
    hash[attr_key] = {
      present: @present.public_send(attr_key),
      future:  @future.public_send(attr_key)
    }
  end
end
```